.gitignore

```diff
@@ -1,11 +1,11 @@
 .idea/
 .vscode/
-*.iml
+.iml
```

Comment:
There is an extra space before the `.iml` entry. It should be removed to ensure consistency.